fuzz/corpora/asn1/00ea60faa14c327b8bc2bc30bfeb4cb8092483a0

```diff
@@ -0,0 +1 @@
+�|
```

Comment:
I wonder why doesn't this count as binary file? I mean why is diff shown, unlike most of the others? If cloned, binary attribute appears to be set...